Gruntfile.coffee

```diff
@@ -43,11 +48,18 @@ module.exports = (grunt) ->
 
   grunt.registerTask 'lint', ['coffeelint']
 
-  grunt.registerTask 'test', [
-    'requirejs:compile'
-    'shell:webdriverManagerUpdate'
-    'connect:server'
-    'protractor:chrome'
-  ]
+  grunt.registerTask 'test', (target) ->
+    return grunt.task.run [
+      'requirejs:compile'
+      'shell:webdriverManagerUpdate'
+      'connect:server'
+      "protractor:#{target}"
+    ]
+
+  grunt.registerTask 'ci', (target) ->
+    return grunt.task.run [
+      'lint'
+      "test:#{target}"
+    ]
 
-  grunt.registerTask 'default', ['lint', 'test']
+  grunt.registerTask 'default', ['ci:firefox']
```

Comment:
I modified the default Grunt task to run against Firefox in order to sidestep the issues detailed in [this thread](https://github.com/travis-ci/travis-ci/issues/272), but support for testing against Chrome remains in other Grunt tasks / npm scripts.